config/memory.go

```diff
@@ -13,8 +13,8 @@ import (
 	"github.com/mattermost/mattermost-server/v5/model"
 )
 
-// memoryStore implements the Store interface. It is meant primarily for testing.
-type memoryStore struct {
+// MemoryStore implements the Store interface. It is meant primarily for testing.
+type MemoryStore struct {
```

Comment:
Is there a reason you have exported this struct ?

Reply:
I exported that to fix these kind of warnings:

```
config/memory.go:35:24: exported func NewMemoryStore returns unexported type *config.memoryStore, which can be annoying to use
config/memory.go:40:62: exported func NewMemoryStoreWithOptions returns unexported type *config.memoryStore, which can be annoying to use
```

There is a discussion and the reasoning is also explained: https://github.com/golang/lint/issues/210

what do you think?

Reply:
Ah I see. That's okay.